File: solutions/yoon-wootak/2026-08-11/medium/1213.java
import java.io.*;
import java.util.*;

class Solution {
    public static void main(String[] args) throws Exception {
        //System.setIn(new FileInputStream("input.txt"));

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();

        for (int test_case=1; test_case<=10; test_case++) {
            br.readLine();
            sb.append("#").append(test_case).append(" ");

            String word = br.readLine();
            String sentence = br.readLine();

            int count = 0;
            for (int i=0; i<=sentence.length()-word.length(); i++) {
                if(sentence.substring(i, i+word.length()).equals(word)) {
                    count++;
                }
            }

            sb.append(count).append("\n");
        }

        System.out.println(sb);
    }
}

File: solutions/kuk0000/2026-08-11/medium/1213.py
T = 10
for test_case in range(1,T+1):
    num = int(input())
    pattern = input().strip()     # 찾을 문자열
    sentence = input().strip()    # 검색할 문장

    N = len(sentence)
    L = len(pattern)
    count = 0

    for i in range(N - L + 1):
        if sentence[i:i+L] == pattern:
            count += 1

    print(f'#{num} {count}')